test/replication/qsync_basic.test.lua

```diff
@@ -247,6 +247,7 @@ test_run:switch('default')
 box.cfg{replication_synchro_timeout = 1000}
 for i = 1, 100 do box.space.sync:replace{i} end
 test_run:cmd('switch replica')
+test_run:wait_lsn('replica', 'default')
```

Comment:
There's `replication_synchro_quorum = 2` left on master from the previous test case (see lines 228, 235).
So `for i = 1, 100 do box.space.sync:replace{i}` will succeed only after the data is really replicated to the replica.

There is no memtx mvcc, so replica should see all 100 entries even without this `wait_lsn`.
If it doesn't, the issue is somewhere else. Not in the missing `wait_lsn`.

Reply:
Never mind, I see now that the test only fails in vinyl. Because vinyl now has 'best-effort' isolation by default, so there are no dirty reads on replica. The fix's correct.